sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/_shared/client_base.py

```diff
@@ -4,7 +4,6 @@
 # ------------------------------------
 from typing import TYPE_CHECKING
 
-from azure.core.configuration import Configuration
```

Comment:
This is used in type checking (so it should be in the `if TYPE_CHECKING` block).

Reply:
ok, moved to there thanks!